### PR TITLE
kinase 0.1.0

### PR DESCRIPTION
**Pull request by indicatelovelace**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This packages allows the easy styling of document links. While Typst produces correct output for internal and external links, there is no easy way to style those. To achieve this the user can set styling rules for different keys. Keys are regex or strings. The package includes regex helpers for mails, phone numbers and urls.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by indicatelovelace** [comment]
> The package is called "kinase", but the PR says "Kinave". which one is correct?

Typo on my end. The package is named after the enzyme kinase. I also changed the title of the pull request to match the case.

File: packages/preview/kinase/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "kinase"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Lennart Schuster"]
+license = "MIT"
+description = "Easy styling for different link types like mails and urls."
+exclude = ["docs/manual.pdf", "ressources/example.png"]
```
[the remark this answers is not in the document]
Comment (indicatelovelace) [reply packages/preview/kinase/0.1.0/typst.toml]: > I think docs & resources could be excluded completely. `ressources` should likely also be `resources` (typo).

Changed it.

[the remark this answers is not in the document]
Comment (elegaanz) [reply packages/preview/kinase/0.1.0/typst.toml]: It still says `ressources` with two `s`, is it intentional?

[the remark this answers is not in the document]
Comment (indicatelovelace) [reply packages/preview/kinase/0.1.0/typst.toml]: Nah english is the only language to spell it that way, I fixed it.

File: packages/preview/kinase/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Kinave
```
Comment (elegaanz) [inline packages/preview/kinase/README.md:1]: There should only be a sub-folder per version in this directory, so I think you should delete this file.

**Comment by indicatelovelace** [comment]
The issues should be fixed, thanks for being so thorough.

**Comment by elegaanz** [comment]
Thank you :)

**Pull request merged**